x.py

```diff
@@ -257,6 +261,55 @@ def ide(args):
     """Start VS Code with the given arguments."""
     run_command(['code'] + args)
 
+def run_benchmarks(args):
+    warmup_iterations = 3
+    bench_iterations = 10
+    results = {}
+
+    report("running benchmarks")
+    env = get_env()
+    report("Starting prusti-server")
+    server_process = subprocess.Popen(["./target/release/prusti-server-driver","--port","12345"], env=env)
```

Comment:
The `prusti-server-driver` path is probably platform-specific, so would be good to add a way to support multiple OS versions.

Reply:
Also, the binary to be executed should be `prusti-server` and it should not need to set environment variables.

Reply:
@fpoli How do you kill it then properly? The kill like below kills `prusti-server`, but `prusti-server-driver` keeps running.

Reply:
Oh, I didn't notice that. `prusti-server` should propagate SIGINT to `prusti-server-driver` or do something to shutdown properly. I opened issue #249. It's also possible that `z3` instances keeps running, even when using `prusti-server-driver`.